index.html

```diff
@@ -8,8 +8,6 @@
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-    <header>
-        <h1>Hello Microcerse</h1>
-    </header>
+    <h1>Hello Microcerse</h1>
```

Comment:
- [ ] please kindly spell `Microverse` correctly. It is important to show recruiters and users of your website that you pay alot of attention to details :+1: